tools/path_fix.sh

```diff
@@ -7,14 +7,14 @@
 #
 # NOTE: This implementation is far from perfect and will need to be refined to cover all cases.
 
-set -e
```

Comment:
I don't feel strongly about this, but shouldn't this tool fix be a separate PR?  Does this have anything to do with the change?

Reply:
I forgot this was in here. I will do it in a different PR with more comments.